__tests__/src/lib/ThumbnailFactory.test.js

```diff
@@ -1,155 +1,356 @@
-import { ManifestResource, Resource, Utils } from 'manifesto.js/dist-esmodule';
-import getThumbnail from '../../../src/lib/ThumbnailFactory';
+import { Canvas, Service, Utils } from 'manifesto.js/dist-esmodule';
+import ThumbnailFactory from '../../../src/lib/ThumbnailFactory';
 import fixture from '../../fixtures/version-2/019.json';
 
-const manifest = Utils.parseManifest(fixture);
```

Comment:
One thing as a reviewer would help me would be to see the added changes separately from the changes of the underlying test suite. Would it be possible to extract only the modified pieces needed for the new functionality from here?

Reply:
sure, I can work on creating a new PR for the rewrite of the test file